src/main/java/Assignment_3/FamilyTreeControllers.java

```diff
@@ -0,0 +1,120 @@
+package Assignment_3;
+
+import java.util.Scanner;
+
+public class FamilyTreeControllers {
+
+  // Print Input Options
+  static void printOptions() {
+    System.out.println("Enter Choice no.");
+    System.out.println("1. Get the immediate parents of a node.");
+    System.out.println("2. Get the immediate children of a node.");
+    System.out.println("3. Get the ancestors of a node.");
+    System.out.println("4. Get the descendants of a node.");
+    System.out.println("5. Delete dependency from a tree.");
+    System.out.println("6. Delete a node from a tree.");
+    System.out.println("7. Add a new dependency to a tree.");
+    System.out.println("8. Add a new node to tree.");
+    System.out.println("9. Exit");
+  }
+
+  // Validate Input NodeId Format
+  static boolean validateInputNodeId(String Input) {
+    if (Input.isEmpty() || !Input.matches("[0-9]*")) {
+      System.out.println("Invalid Input");
+      return false;
+    }
+    return true;
+  }
+
+  public static void main(String[] args) {
+    // Exit flag
+    boolean exit = false;
+    FamilyTreeService familyTreeService = new FamilyTreeService();
+    try {
+      while (!exit) {
+        printOptions();
+        Scanner sc = new Scanner(System.in);
+        String ChoiceString = sc.nextLine().trim();
```

Comment:
there are multiple places, where naming convention is not proper, correct at all places